src/domain/venues/query/VenueQueries.ts

```diff
@@ -6,10 +6,23 @@ export const venuesQuery = gql`
       edges {
         node {
           id
+          project {
+            id
+            translations {
+              languageCode
+              name
+            }
+          }
```

Comment:
Why do we need to fetch project here?

Reply:
Old habit of mine, fetching all the data to make sure that we always have the most recent information available. And at some point, we might want to organise venues by project in case we have super-admins or multi-project admins.  

If you think this causes more problems than it theoretically could solve I. can remove it